src/Model/Constants.cs

```diff
@@ -127,44 +92,11 @@ internal static class Constants
             "OutBuffer",
             "OutVariable",
             "PipelineVariable",
+            "ProgressAction",
```

Comment:
This is PS v7+ only. Should we distinguish between WinPS v/s v7+

Reply:
yah, i think we probably should. Not sure how we'll know though.

Reply:
We don't really have a good solution for this (I talked to Sean about it). For now, I'm going to included it.